financialaid/serializers.py

```diff
@@ -109,7 +109,7 @@ def validate(self, data):
                 self.instance.status != FinancialAidStatus.PENDING_MANUAL_APPROVAL):
             raise ValidationError("Cannot approve application that is not pending manual approval.")
         if (data['action'] == FinancialAidStatus.PENDING_MANUAL_APPROVAL and
-                self.instance.status != FinancialAidStatus.PENDING_DOCS):
+                self.instance.status not in [FinancialAidStatus.PENDING_DOCS, FinancialAidStatus.DOCS_SENT]):
             raise ValidationError("Cannot mark documents as received for application not pending docs.")
```

Comment:
Since you're much more familiar with the front end and what users might see, if you have better messages, we're happy to put them in!